pages/administration/tools/booking-lookup.vue

```diff
@@ -0,0 +1,327 @@
+<template>
+  <AdminPage title="Bookings Lookup">
+    <div class="flex items-end space-x-4">
+      <div>
+        <label>User</label>
+        <UiInputText
+            v-model="userSearch"
+                     placeholder="Filter by user" />
+      </div>
+      <div>
+        <!-- Show the productions filter if we have any. If not, disable the
+        filter.-->
+        <label>Production</label>
+        <UiInputSelect
+            v-if="!$apollo.queries.productions.loading && productions.length
+            > 0"
+            v-model="productionSlug"
+            nullify-value="performanceId"
+            :options="productions"
+        />
```

Comment:
I wonder if we should have a more effective way to display productions that isn't a list that will grow incredibly long over time. I know you touch on this in the PR's notes.

I've wracked my brains, and perhaps some combo search/select tool might be most useful. This would allow you to search via free text, and if more than one production matches your text search it offers you a list of these productions alongside the date range of its performances, so that you don't have to trawl through three `TRASh`s manually to find this year's one. However, very open to other ideas!